log/helper.go

```diff
@@ -75,6 +75,11 @@ func (h *Helper) Enabled(level Level) bool {
 	return true
 }
 
+// Logger returns logger in the helper.
+func (h *Helper) Logger() Logger {
```

Comment:
please supplement unit testing and example

Reply:
> please supplement unit testing and example

@shenqidebaozi TestCase for Logger has been provide in [this commit](https://github.com/go-kratos/kratos/pull/3443/commits/2dcf06b0bb712ee85af889322ca6ebb8a7fee09b)